full/src/main/java/apoc/load/LoadDirectoryItem.java

```diff
@@ -0,0 +1,127 @@
+package apoc.load;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class LoadDirectoryItem {
+
+    public static final String EVENT_KINDS = "eventKinds";
+    public static final String INTERVAL = "interval";
+    public static final List<String> DEFAULT_EVENT_KINDS = List.of("ENTRY_CREATE", "ENTRY_DELETE", "ENTRY_MODIFY");
+
+    public static class LoadDirectoryConfig {
+
+        private final List<String> eventKinds;
+        private final Long interval;
+
+        public LoadDirectoryConfig(Map<String, Object> config) {
+            if (config == null) config = Collections.emptyMap();
+            this.interval = (Long) config.getOrDefault("interval", 1000L);
+            this.eventKinds = (List<String>) config.getOrDefault("eventKinds", DEFAULT_EVENT_KINDS);
```

Comment:
fai una cosa questo modificalo in modo che:
* `eventKind` diventa `listenEventType`
* i valori ammessi sono `[CREATE, DELETE, MODIFY]` dopo di che nella config ti creai una `List<Kind>`
* trasforma `DEFAULT_EVENT_KINDS` in Set<Kind>